modules/Rock.cmake

```diff
@@ -108,9 +108,16 @@ endfunction()
 
 
 ## Main initialization for Rock CMake projects
-macro (rock_init PROJECT_NAME PROJECT_VERSION)
-    project(${PROJECT_NAME})
-    set(PROJECT_VERSION ${PROJECT_VERSION})
+macro (rock_init)
+    if (${ARGC} GREATER 0)
+        message(WARNING "Passing project name and version to rock_init was a misfeature")
```

Comment:
How about checking if PROJECT_NAME and/or PROJECT_VERSION are already set, to silence the warning, and be more explicit on how to replace rock_init(project_name version).

Reply:
Because we still want people to stop calling rock_init with arguments. I'm making the warning more explicit.